lib/internal/util/inspect.js

```diff
@@ -1259,6 +1259,49 @@ function removeDuplicateErrorKeys(ctx, keys, err, stack) {
   }
 }
 
+function markNodeModules(ctx, line) {
+  let tempLine = '';
+  let nodeModule;
+  let pos = 0;
+  while (nodeModule = nodeModulesRegExp.exec(line)) {
```

Comment:
```suggestion
  while (nodeModule = RegExpPrototypeExec(nodeModulesRegExp, line)) {
```

Reply:
Thanks for the suggestions. However, I kindly ask not to suggest to add more primordials up until there is a decision how to progress with primordials in general.